legiti/build.gradle

```diff
@@ -9,11 +9,11 @@ apply plugin: 'org.jetbrains.dokka-android'
 apply plugin: 'kotlinx-serialization'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
 
     defaultConfig {
-        minSdkVersion 15
-        targetSdkVersion 28
+        minSdkVersion 18
```

Comment:
We needed this for the MediaDRM. This is still lower than zeenow and ingresse minimum version for the app